content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md

```diff
@@ -0,0 +1,807 @@
+
+- **kind**: Namespace
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../cluster-resources/namespace-v1#NamespaceSpec" >}}">NamespaceSpec</a>)
+<!--
+  Spec defines the behavior of the Namespace. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+  Spec 定义了 Namespace 的行为。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+- **status** (<a href="{{< ref "../cluster-resources/namespace-v1#NamespaceStatus" >}}">NamespaceStatus</a>)
+<!--
+  Status describes the current status of a Namespace. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+  Status 描述了当前 Namespace 的状态。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+## NamespaceSpec {#NamespaceSpec}
+<!--
+NamespaceSpec describes the attributes on a Namespace.
+-->
+NamespaceSpec 用于描述 Namespace 的属性。
+
+<hr>
+
+- **finalizers** ([]string)
+<!--
+  Finalizers is an opaque list of values that must be empty to permanently remove object from storage. More info: https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
+-->
+  Finalizers 是一个不透明的值列表，必须为空时才能从存储中永久删除对象。 更多信息： https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
+
+## NamespaceStatus {#NamespaceStatus}
+<!--
+NamespaceStatus is information about the current status of a Namespace.
+-->
+NamespaceStatus 表示当前 Namespace 的状态信息。
+<hr>
+
+- **conditions** ([]NamespaceCondition)
+<!--
+  *Patch strategy: merge on key `type`*
+  
+  Represents the latest available observations of a namespace's current state.
+-->
+  **补丁策略：基于 `type` 健合并**
+  
+  表示当前最新可用的命名空间状态。
+
+  <a name="NamespaceCondition"></a>
+<!--
+  *NamespaceCondition contains details about state of namespace.*
+
+  - **conditions.status** (string), required
+
+    Status of the condition, one of True, False, Unknown.
+-->
+  **NamespaceCondition 包含命名空间状态的细节。**
+
+  - **conditions.status** (string), 必需
+
+    条件的状态，为True，或False，或Unknown。
+<!--
+  - **conditions.type** (string), required
+
+    Type of namespace controller condition.
+    
+  - **conditions.lastTransitionTime** (Time)
+-->
+  - **conditions.type** (string), 必需
+
+    命名空间控制器条件类型。
+    
+  - **conditions.lastTransitionTime** (Time)
+
+    <a name="Time"></a>
+<!--
+    *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+    **Time 是时间的包装。Time 支持对YAML和JSON进行正确封送。为时间包提供的许多函数方法提供了包装器**
```

Comment:
There should be a space between Mandarin and English word.
Ref: https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F

```suggestion
    **Time 是 time.Time 的包装，支持对 YAML 和 JSON 进行正确封送，为 time 包的许多函数方法提供了包装器。**
```